src/clean_acquisitions_data.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+import re
+import argparse
+
+import yaml
+import pandas as pd
+import boto3
+
+from helpers import gen_helpers as gen_h
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+def get_acquisition_count(df, all_companies_file):
+    """Aggregates acquisition by acquirer and creates new dataframe for
+    acquisition count"""
+    # Create list of acquired companies in original company dataset
+    company_df = gen_h.read_data(all_companies_file)
+    all_companies_list = list(company_df['permalink'].unique())
+    acquired_companies_list = list(df['company_permalink'].unique())
+    acquirer_companies_list = list(df['acquirer_permalink'].unique())
+
+    acquired_companies_list = [permalink for permalink in acquired_companies_list
+    if permalink in all_companies_list]
+    acquirer_companies_list = [permalink for permalink in acquirer_companies_list
+    if permalink in all_companies_list]
+
+    # Create dictionary of number of acquisitions by acquirers that are in original company dataset
+    acquirer_dict = dict(df['acquirer_permalink'].value_counts())
+    acquirer_dict = {acquirer: acquirer_dict[acquirer] for acquirer in acquirer_dict
+    if acquirer in acquirer_companies_list}
+
+    # Create dataframe of number of acquisitions to merge into other dataframes
+    acquirers_df = pd.DataFrame.from_dict(acquirer_dict, orient='index').reset_index()
```

Comment:
This is so that the object returned from this script would be a df, be save-able, and merge-able with other dataframes in the later preprocess merge script

Reply:
but why not keep as df all the way? you do a to_dict then from_dict